src/input_common.cpp

```diff
@@ -270,3 +270,125 @@ void input_event_queue_t::prepare_to_select() {}
 void input_event_queue_t::select_interrupted() {}
 void input_event_queue_t::uvar_change_notified() {}
 input_event_queue_t::~input_event_queue_t() = default;
+
+readline_cmd_t::readline_cmd_t(const readline_cmd_t& rl) : id_(rl.id_) {
```

Comment:
we still use old-school pointer alignment (`readline_cmd_t &code`), `clang-format` should do the trick

Reply:
Will do. I think `clang-format` chooses whatever is most common in the file unless one sets
```
DerivePointerAlignment: false
PointerAlignment: Right
ReferenceAlignment: Pointer
```
in `.clang-format`